src/wui/interactive_player.cc

```diff
@@ -571,14 +571,22 @@ void InteractivePlayer::draw_map_view(MapView* given_map_view, RenderTarget* dst
 
 			const Image* player_image = nullptr;
 			float icon_scale = 0.7f;
+			float icon_opacity = 1.0f;
 
 			// Not all map starting positions pass the suitability test.
 			// TODO(tothxa): Make the editor at least use the same test. But manual changes would still
 			//               be possible.
 			for (unsigned p = map.get_nrplayers(); p != 0u; --p) {
 				if (map.get_starting_pos(p) == f->fcoords) {
-					player_image = playercolor_image(p - 1, icon_filename);
+					Widelands::Player* plr = gbase.get_player(p);
+					if (plr == nullptr || !plr->is_picking_custom_starting_position()) {
+						// Should have a HQ if finished picking, no need for the overlay
+						continue;
+					}
+					player_image = playercolor_image(plr->get_playercolor(), icon_filename);
 					icon_scale = 1.0f;
+					icon_opacity =
+					   plr->get_starting_position_suitability(f->fcoords) ? 0.7f : kBuildhelpOpacity;
```

Comment:
I understand:
- opacity is 0.7 for suitable starting position
- it is kBuildhelpOpacity (0.3 from line 498) for not suitable starting position

Does 0.7 look better than 1.0 from before?

Reply:
I only wanted to give it some transparency because this is not the actual starting position. Maybe 0.7 is still too opaque though.

Reply:
> I only wanted to give it some transparency because this is not the actual starting position.

Good reason.

> Maybe 0.7 is still too opaque though.

No idea, I have not seen it. Feel free to decide.